<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>10</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #1a1a2e,
        stop: 1 #16213e
    );
    border-radius: 10px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Screen.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #ff512f,
        stop: 1 #dd2476
    );
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #dd2476,
        stop: 1 #ff512f
    );
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #cc1c66,
        stop: 1 #ee4025
    );
}
</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #1a1a2e,
        stop: 1 #16213e
    );
    border-radius: 10px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/second.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="StartGameButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>480</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #ff512f,
        stop: 1 #dd2476
    );
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #dd2476,
        stop: 1 #ff512f
    );
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #cc1c66,
        stop: 1 #ee4025
    );
}
</string>
     </property>
     <property name="text">
      <string>Start An Async PVP Match</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>400</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #ff512f,
        stop: 1 #dd2476
    );
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #dd2476,
        stop: 1 #ff512f
    );
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #cc1c66,
        stop: 1 #ee4025
    );
}
</string>
     </property>
     <property name="text">
      <string>Start a Async PVP Match</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>781</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #1a1a2e,
        stop: 1 #16213e
    );
    border-radius: 10px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/galler.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>460</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #ff512f,
        stop: 1 #dd2476
    );
    color: white;
    border: 2px solid white;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #dd2476,
        stop: 1 #ff512f
    );
}

QPushButton:pressed {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #cc1c66,
        stop: 1 #ee4025
    );
}
</string>
     </property>
     <property name="text">
      <string>BAck</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/first Fighter.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/second Fighter.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>70</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/third Fighter.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>first Player</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Second Player</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_player1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_player2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
